CHANGELOG/CHANGELOG-1.7.md

```diff
@@ -0,0 +1,110 @@
+  * Add an Antrea Agent configuration option `multicast.igmpQueryInterval` to specify the interval at which the antrea-agent sends IGMP queries to Pods. ([#3819](https://github.com/antrea-io/antrea/pull/3819), [@liu4480])
+- Add the following capabilities to the Multicluster feature:
+  * Add a number of `antctl mc` subcommands for bootstrapping Multi-cluster; refer to [antct.md](https://github.com/antrea-io/antrea/blob/release-1.7/docs/multicluster/antctl.md) for more information. ([#3474](https://github.com/antrea-io/antrea/pull/3474), [@hjiajing])
+  * <Placeholder for Gateway, ClusterInfo related changes>. ([#3689](https://github.com/antrea-io/antrea/pull/3689) [#3463](https://github.com/antrea-io/antrea/pull/3463) [#3603](https://github.com/antrea-io/antrea/pull/3603), [@luolanzone])
+- Add the following capabilities to secondary network IPAM:
+  * Add support for multiple IPPools. ([#3606](https://github.com/antrea-io/antrea/pull/3606), [@jianjuns])
+  * Add support for static addresses. ([#3633](https://github.com/antrea-io/antrea/pull/3633), [@jianjuns])
+  * Add support for IPAM for Pod secondary networks managed by Multus. ([#3529](https://github.com/antrea-io/antrea/pull/3529), [@jianjuns])
+- [Windows] Add support for NodePortLocal on Windows Nodes. ([#3453](https://github.com/antrea-io/antrea/pull/3453), [@XinShuYang])
+- [Windows] Add support for Traceflow on Windows Nodes. ([#3022](https://github.com/antrea-io/antrea/pull/3022), [@gran-vmv])
+- Add support for containerd to antrea-eks-node-init.yml. ([#3840](https://github.com/antrea-io/antrea/pull/3840), [@antoninbas])
+- Add an Antrea Agent configuration option `disableTXChecksumOffload` to support cases in which the datapath's TX checksum offloading does not work properly. ([#3832](https://github.com/antrea-io/antrea/pull/3832), [@tnqn])
+- Support Internal Traffic Policy for AntreaProxy. ([#2792](https://github.com/antrea-io/antrea/pull/2792), [@hongliangl])
+- Add the following documentations:
+  * Add [documentation](https://github.com/antrea-io/antrea/blob/release-1.7/docs/security.md#protecting-your-cluster-against-privilege-escalations) for the Antrea Agent RBAC permissions and how to restrict them using Gatekeeper/OPA. ([#3694](https://github.com/antrea-io/antrea/pull/3694), [@antoninbas])
+  * Add information about Theia in [network-flow-visibility](https://github.com/antrea-io/antrea/blob/release-1.7/docs/network-flow-visibility.md) doc. ([#3886](https://github.com/antrea-io/antrea/pull/3886), [@jianjuns])
+  * Add [quick start guide](https://github.com/antrea-io/antrea/blob/release-1.7/docs/multicluster/quick-start.md) for Antrea Multi-cluster. ([#3853](https://github.com/antrea-io/antrea/pull/3853), [@luolanzone])
```

Comment:
Probably add @jianjuns too (I co-authored the doc with Lan).

Reply:
done